src/GeneralStateTestsFiller/stEWASMTests/utils/callDataCopyFiller.yml

```diff
@@ -0,0 +1,67 @@
+callDataCopy:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    # tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+    # main contract, tx receiver
+    abcdabcdabcdabcdabcdabcdabcdabcdabcdabcd:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "callDataCopy" (func $callDataCopy (param i32 i32 i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (import "debug" "printMemHex" (func $printMemHex (param i32 i32)))
```

Comment:
IMO let's try and move debug statements out of tests after we have ensured that the WAST is written properly. The code is easier to read and understand when there aren't debug statements in it.

Reply:
Agreed, good catch, will fix